packages/gatsby-theme-newrelic/src/utils/isLocalStorageAvailable.js

```diff
@@ -0,0 +1,12 @@
+const isLocalStorageAvailable = () => {
```

Comment:
I'm not 100% sure, but this might be simpler as `'setItem' in localStorage && 'getItem' in localStorage`

Reply:
I like it! Although it looks like the try/catch are still required because certain browsers (looking at you, Brave) throw an error instead of just removing localStorage from the global scope.